tests/test_stub_attribution_organic.py

```diff
@@ -54,12 +54,14 @@ def assert_good(new_dict, source, medium, campaign, content):
     assert new_dict == old_dict
 
 
-@pytest.mark.parametrize('source, medium, campaign, content', [
-    ('www.allizom.org', 'referral', '(not set)', '(not set)')])
-def test_organic_flow_param_values(base_url, selenium, source, medium, campaign, content):
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('medium, campaign, content', [
+    ('referral', '(not set)', '(not set)')])
```

Comment:
As mentioned during our hack session, parameterised tests are most valuable when there's at least two sets of values. If your tests will never need to pass additional values then we can probably just hard-code them.